openvdb_cmd/vdb_tool/CMakeLists.txt

```diff
@@ -0,0 +1,159 @@
+cmake_minimum_required(VERSION 3.10)
+
+# CMP0091 allows for MSVC ABI targetting via CMAKE_MSVC_RUNTIME_LIBRARY
+# from CMake 3.15 and above. Must come before project().
+if(POLICY CMP0091)
+  cmake_policy(SET CMP0091 NEW)
+endif()
+
+if(POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
+endif()
+
+project(vdb_tool LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Remove

Reply:
I'll comment these out (and remove them entirely later)